Module 15 - Lightning Learning/lwc/createContactLWC/createContactLWC.js

```diff
@@ -0,0 +1,35 @@
+import { LightningElement, api } from 'lwc';
+import { ShowToastEvent } from 'lightning/platformShowToastEvent';
+
+import NAME_FIELD from '@salesforce/schema/Contact.Name';
+import EMAIL_FIELD from '@salesforce/schema/Contact.Email';
+import PHONE_FIELD from '@salesforce/schema/Contact.Phone';
+import FAX_FIELD from '@salesforce/schema/Contact.Fax';
+
+export default class createContactRecord extends LightningElement
```

Comment:
class name should start with capital case.
class name should be same as file name Always